### PR TITLE
Tailwind Refresh

### PR DESCRIPTION
**Pull request by ericnewcomer**
### ToDo
- [x] Welcome landing
- [x] Selection for list pages only by checkbox, row-click leaves or edits
- [x] Login
- [x] Invitations (@norkans7)
- [x] Channels (@ericnewcomer)
  - [x] Claim numbers
  - [x] Android senders / callers
  - [x] Add pipeline
  - [x] Detail page
  - [x] Settings
  - [x] Channel claim list
- [x] Invert list selection rules
- [x] Legacy editor
- [x] Globals (@norkans7)
- [x] Tickets (@rowanseymour)
- [x] Confirmation for changing flow base language
- [x] Contact history doesn't look amazing (closer to biz example)
- [x] Contact featured fields
- [x] Download flow results (not export) (@norkans7)
- [x] Run results
- [x] Campaign event deletion
- [x] Navigation icons
- [x] Logs (@norkans7)
  - [x] Webhooks 
  - [x] Channels
  - [x] Flow starts 
- [x] Message schedules
- [x] Trigger schedules
- [x] Unit test failures (@norkans7)
- [x] Policy
- [x] API (@norkans7)
- [x] Org Import page  (@norkans7)
- [x] Org Export page  (@norkans7)
- [x] Contact import page  (@ericnewcomer)
- [x] Manage logins (@norkans7)
- [x] Safari not laying out outer frame properly
- [x] Create Account (@ericnewcomer)
- [x] Billing (@ericnewcomer)
- [x] Modal forms should disable / mask when submitting (@ericnewcomer)
- [x] Featured field / pinned / show in table consistency (@ericnewcomer)
- [x] Formax form failure hides summary (@ericnewcomer)
- [x] Enter on text box should submit form (@ericnewcomer)
- [x] Text input insertion broken
- [x] Classifiers

### CODE REVIEW

File: templates/vanilla/frame.haml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+<!--[if !IE]><!--><html lang="en">              <!--<![endif]-->
+{% endblock %}
+  
+  %head
+    %title
+      {% block page-title %}
+        {{ brand.name }} - Visually build your own chatbots
+      {% endblock %}
+
+    %meta{charset:"utf-8"}
```
Comment (rowanseymour) [inline templates/vanilla/frame.haml:14]: if you're writing new templates.. don't forget you can use the newer syntax, e.g. `%meta(charset="utf-8")`

**Comment by rowanseymour** [comment]
@ericnewcomer whats the difference between `static/fonts/icons` and `static/fonts/temba`?

**Comment by ericnewcomer** [comment]
I'm refreshing the icons and giving them useful names in the process. Once we are fully refreshed the old icon set will go away.

**Comment by rowanseymour** [comment]
Ah ok. We also have `static/icons`.. not sure if anything is using that

**Comment by ericnewcomer** [comment]
I'll have a look at those during this cleanup

File: templates/vanilla/frame.haml (hunk 1 above)
Comment (ericnewcomer) [reply templates/vanilla/frame.haml:14]: so, i'm aware i'm allowed to, but what's the benefit?

Comment (rowanseymour) [reply templates/vanilla/frame.haml:14]: well we've generally been using it when we write new templates.. I thought we agreed it was nicer, e.g.

```haml
%meta{name:"author", content:"John Doe"}
```

vs

```haml
%meta(name="author" content="John Doe")
```

I mostly like that it doesn't need commas and looks more like html

**Comment by ericnewcomer** [comment]
Just realized I was doing my refresh work on this branch, so marking it draft and trucking along.

**Comment by nicpottier** [comment]
Woot,  nice work @ericnewcomer and team!

File: karma/helpers.coffee [hunk 2]
```diff
@@ -1,5 +1,7 @@
 fetchPJAXContent = (url, container, options) ->
+  console.log("fetchPJAXContent")
   type = 'GET'
+  bleep = 'Bloop'
```
Comment (rowanseymour) [inline karma/helpers.coffee:4]: ?

File: karma/helpers.coffee [hunk 3]
```diff
@@ -1,5 +1,7 @@
 fetchPJAXContent = (url, container, options) ->
+  console.log("fetchPJAXContent")
```
Comment (rowanseymour) [inline karma/helpers.coffee:2]: crud

File: temba/channels/views.py [hunk 4]
```diff
@@ -1542,7 +1604,7 @@ class DomainForm(forms.Form):
             whitelisted_domain = forms.URLField(
                 required=True,
                 initial="https://",
-                help_text="The domain to whitelist for Messenger extensions  ex: https://yourdomain.com",
+                help_text="The domain to whitelist for Messenger extensions ex: https://yourdomain.com",
```
Comment (rowanseymour) [inline temba/channels/views.py:1607]: `_()`

**Pull request merged**